scripts/init.d-xapi-domains

```diff
@@ -6,6 +6,12 @@
 # chkconfig: 345 99 00
 # description: Start/stop XE VMs
 
+### BEGIN INIT INFO
+# Required-Start:    $remote_fs
+# Short-Description: Start/stop XE VMs
+# Description: Stop domains automatically when domain 0 shuts down.
```

Comment:
Without sounding stupid, I guess these are not commented out? ;) Presumably these are "special".